src/rsz/test/make_parasitics7_ss.spefok

```diff
@@ -0,0 +1,122 @@
+*SPEF "ieee 1481-1999"
```

Comment:
How come there are two spefok files but you are only checking one of them.
Second why would you write the ss file if there is only one corner defined?

Reply:
It was not intended to be in this PR, I added this file by accident. Thanks for noting that!

Reply:
Ahh, okay.